wiki/markdown/step-by-step-subscription-notification.md

```diff
@@ -165,20 +165,20 @@ complete notification message can be seen below:
         runpipeline: mybuildstep
 
 
-## Missed notifications
+## Failed notifications
 
 If the notification via REST POST fails, then a fixed number of attempts are
 made to resend successfully. The number of attempts are specified in the
 [application.properties](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/main/resources/application.properties)
 as “notification.failAttempt”. If message sending attempts fails for the
-specified number of time, then a missed notification is prepared and stored
+specified number of time, then a failed notification is prepared and stored
 in database. The name of the database is specified in the application.properties
 file as “missedNotificationDataBaseName” and collection name as
```

Comment:
Is this property still named "missedNotificationDataBaseName" ? or something with "failed-notification........"

Reply:
Now, I have changed it to failedNotification...

Reply:
Changing the property name is not really part of the original issue, since we only referred to changing the interface. Updating property name will **affect users and this needs to be reflected in the pull request so we can create good release notes** later. Also any local variables still used in the code are still named missed notification something and needs to be updated in that case. Maybe we should have a separate pull request for this?

Reply:
How it will effect the user...?

Reply:
If the configuraiton properties change names, e.g.: missedNotificationDataBaseName -->failedNotificationDataBaseName like @tobiasake suggested then it will affect existing users. If they want to use the latest release they will need to update the configuration files they use to adapt to these new properties. (Next release will be non backwards compatible with these REST endpoints anyway, but we still need to inform users of what has changed)

Reply:
Well, documentation is. anyway, being updated, i.e., from missedNotification to failedNotifiction. And, it is quite natural to change the properties so little confusion prevails regarding the name of the database and endpoint. It is quite logical to change the property name and endpoint name at the same time as db name reflects endpoint name.

Moreover, user will effect little rather developer will be facilitated by the similar names.

Reply:
Have you updated the properties with a more proper name?

Reply:
yes